include/sdf/Geometry.hh

```diff
@@ -56,6 +57,9 @@ namespace sdf
 
     /// \brief A mesh geometry.
     MESH = 5,
+
+    /// \brief An ellipsoid geometry
+    ELLIPSOID = 6,
```

Comment:
I think this will conflict with the `HEIGHTMAP` value proposed in #388 (targeting `libsdformat9`); that's why I used `7` for `CAPSULE` in #389

Reply:
Ahh, that makes sense. Setting it to 8 in latest commit.